docs/src/quill/ecs.md

```diff
@@ -0,0 +1,31 @@
+# ECS
+ECS stands for Entity Component System and is an integral part of feather and quill.
+
+ECS is an alternative to Object Oriented Programming where you usally stores data in a object and then arrays of the object.
+In ecs you store each field as an array and combine the data into an object at the end. 
+
+
+A `Component` is either a marker or a small amount of data.
+An `Entity` is a set of components.
```

Comment:
Maybe say that an `Entity` is just an ID that can be used to access components, rather than the actual set of components (which are stored in the `Ecs`).